Makefile

```diff
@@ -262,6 +263,8 @@ endif
 
 #### Main Targets ###
 
+all: rom compressed
```

Comment:
```suggestion
all: rom compress
```
(or move this back imo it's just diff/blame noise but whatever)

Reply:
the other PR won so I merged main into this one

Reply:
Oh, I didn't realize I messed it up. ty